src/sidebar/app.js

```diff
@@ -37,6 +38,10 @@ if (settings.raven) {
   angular.module('ngRaven', []);
 }
 
+if(settings.googleAnalytics){
```

Comment:
yeah, this is consistent with the rest of the file as it is right now. So I am going to leave that up to eslint --fix later on